modules/report/view/altinn_report1.php

```diff
@@ -0,0 +1,867 @@
+<?
```

Comment:
Rename file to altinn_yearly_report_amelding.php and link to that one in report.list view